src/Symfony/Component/Console/Application.php

```diff
@@ -61,7 +61,7 @@ class Application
     private $autoExit;
     private $definition;
     private $helperSet;
-    private $dispatcher;
+    protected $dispatcher;
```

Comment:
The container is available in [Symfony fullstack](https://github.com/symfony/symfony/blob/v3.0.6/src/Symfony/Bundle/FrameworkBundle/Console/Application.php#L70), not when the Console component is used alone.